community/modules/remote-desktop/chrome-remote-desktop/scripts/configure-grid-drivers.yml

```diff
@@ -88,8 +102,10 @@
     - name: Install GPU driver
       ansible.builtin.shell: |
         #jinja2: trim_blocks: "True"
-        {% if ansible_distribution_release == "jammy" %}
+        {% if ansible_distribution_release == "jammy"  %}
         CC=gcc-12 /tmp/NVIDIA-Linux-x86_64-525.125.06-grid.run --silent
+        {% elif ansible_distribution_release == "bookworm" %}
+        /tmp/NVIDIA-Linux-x86_64-525.105.17-grid.run --silent
```

Comment:
Can we find a way to not have these hard coded twice. I would suggest creating a dictionary with the part of the variable path (`vGPU14.2/NVIDIA-Linux-x86_64-510.85.02-grid.run`) and then use that in get and install. See how packages is done. To include `vGPU14.2` part you will have to use dest: /tmp/{path name}.

Another option would be to specify the same dest for each driver but I think I like leaving a breadcrumb of what driver was installed (and not renaming the file).

Reply:
Done